    @Test
    public void testModuloInvalidInitNoArgs() {
        try {
            ff.function(functionName);
        } catch (RuntimeException e) {
            Assert.assertEquals("Unable to find function " + functionName, e.getMessage());
            return;
        }

        Assert.fail("Exception not thrown");
    }

